From: Sergei Shtylyov <sergei.shtylyov@cogentembedded.com>
Subject: ARM: dts: porter: remove enable prop from HS-USB device node

In  the final versions of the Porter board (called "PORTER_C") Renesas
decided to get rid  of the Maxim Integrated MAX3355 OTG chip and didn't
add any other provision to differ the host/gadget mode, so we'll have to
remove  no longer valid "renesas,enable-gpio" property from the HS-USB
device node.  Hopefully, the earlier revisions of the board were never
seen in the wild...

Fixes: c794f6a09a25 ("ARM: shmobile: porter: add HS-USB DT support")
Reported-by: Yoshihiro Shimoda <yoshihiro.shimoda.uh@renesas.com>
Signed-off-by: Sergei Shtylyov <sergei.shtylyov@cogentembedded.com>
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>

---
 arch/arm/boot/dts/r8a7791-porter.dts |    1 -
 1 file changed, 1 deletion(-)

Index: linux-stable/arch/arm/boot/dts/r8a7791-porter.dts
===================================================================
--- linux-stable.orig/arch/arm/boot/dts/r8a7791-porter.dts
+++ linux-stable/arch/arm/boot/dts/r8a7791-porter.dts
@@ -274,7 +274,6 @@
 	pinctrl-names = "default";
 
 	status = "okay";
-	renesas,enable-gpio = <&gpio5 31 GPIO_ACTIVE_HIGH>;
 };
 
 &usbphy {
